rush.json

```diff
@@ -138,6 +138,12 @@
       ]
     },
     {
+      "packageName": "@microsoft/decorators",
+      "projectFolder": "libraries/decorators",
+      "reviewCategory": "libraries",
+      "shouldPublish": false
```

Comment:
FYI shouldPublish=false here because this is also a browser library, so we need to sort out the manifest deployment before we start publishing this.